README.md

```diff
@@ -87,10 +87,10 @@ pydocstyle iati/
 
 Alternatively, the Makefile can be used:
```

Comment:
Makefile must be used at base project level (`IATI-Standard-Website`), whereas pytest must be executed at Django project root level (`IATI-Standard-Website/iati`)